app/cinema/bar.py

```diff
@@ -0,0 +1,5 @@
+class CinemaBar:
+
+    @staticmethod
+    def sell_product(product: str, customer: str) -> None:
```

Comment:
The method accepts an instance of the Customer class, not a string.
```suggestion
    def sell_product(product: str, customer: Customer) -> None:
```